infrastructure/helm/keycloak/keycloak-setup.md

```diff
@@ -0,0 +1,64 @@
+# Install Quantumserverless and Configure Keycloak
+## In the command line
+Switch default namespace to the install target namespace
+```
+kubectl config set-context –current –namespace="target namespace"
+```
+Install helm chart
+```
+helm install "release name" quantemserverless
+```
+Wati for all pods come up except `releasename-kuberay-head-xxxx`
+Run ./login-setup.sh <RELEASE NAME>
+
+## In the browser
+# Configure Keycloak
+
+Open browser at http://<LOCALIP>:<KEYCLOAK PORT>
+ - Select Administration Console
+ - Login with “user” / <PASSWORD>
+  <LOCALIP>, <KEYCLOAK PORT>, <PASSWORD> are in the login-setup.sh script output.
+
+Create “quantumserverless” Realm
+ - Dropdown “master”
+  -- “Create Realm”
+  -- Set “Realm name” “quantumserverless”
+  -- “Create”
+ - "Realm settings" in the left Nav
+  -- "Action"
+  -- "Partial import"
+  -- "Browse"
+  -- Select "keycloakconfig.json"
+  -- Check all resource check boxes and select Overwrite
+  -- "Import"
+  -- "Close"
+
+Generate “rayclient” client secret
+ - “Clients” in the left nav
```

Comment:
Can this be done through configuration too? Like we are doing [here](https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/infrastructure/helm/quantumserverless/values.yaml#L39), for example.

Reply:
The client secret can be embedded in the import file of the realm.